### DISCUSSION TITLE
clamp sensor PJ-1203A lets my database explode

### DISCUSSION THREAD

**Discussion by hagagit** [Koenkk/zigbee2mqtt#21698] [Devices]
The sensor exposes 17 measurements, and each of this measurements has all the other measurements as state_attribites. And that every 10s. Thats a lot. I still do not understand  how this can generate ~500MB database data each day, but as soon as I exclude this sensor from the recorder this increase of the database is gone. Why does each measurement have all the other measurements as state_attrubutes?

**Comment by schauveau** [comment]
Indeed. The PJ-1203A sends multiple zigbee message to update each of the values and each one generates a MQTT message with the full state ; So something like 15 packets in 1 seconds followed by a pause of 10s. 

A simple solution is to set `debounce: 1` to wait 1 second after the last zigbee message before sending the MQTT message. See also https://www.zigbee2mqtt.io/guide/configuration/devices-groups.html#common-device-options

You may also want to have a look at the alternative converter given in https://github.com/Koenkk/zigbee2mqtt/issues/18419#issuecomment-1983353506
It provides a configuration option to set the delay between the PJ-1203A updates (from 3 to 60s).

**Comment by schauveau** [comment]
Also, I just noticed that `last_seen` (in the global advanced settings) is messing up with `debounce`.  Make sure that it remains disabled. 

See https://github.com/Koenkk/zigbee2mqtt/issues/21953
